elements/nuxeo-results/nuxeo-results.js

```diff
@@ -376,6 +434,16 @@ Polymer({
     );
   },
 
+  _displayDelegatedAction() {
+    return this._displaySelectAll() || this._displaySort();
+  },
+
+  _displaySelectAll() {
+    return (
+      this.view && !this.view.handlesSelectAll && (this.view.hasAttribute('select-all-enabled') || hasSelectAllEnabled)
```

Comment:
Wouldn't this be the same as:
```suggestion
      this.view && !this.view.handlesSelectAll && (this.view.hasAttribute('select-all-enabled') || this.selectAllEnabled)
```
?

Reply:
⬆️ as discussed above.